src/docker.go

```
@@ -1,676 +1,676 @@
+		fmt.Printf("获取layer内容失败: %v\n", err)
+		c.String(http.StatusInternalServerError, "Failed to get layer content")
+		return
+	}
+	defer reader.Close()
+
+	c.Header("Content-Type", "application/octet-stream")
+	c.Header("Content-Length", fmt.Sprintf("%d", size))
+	c.Header("Docker-Content-Digest", digest)
+
+	c.Status(http.StatusOK)
+	io.Copy(c.Writer, reader)
```

golangci-lint (1.64.8)

Error return value of `io.Copy` is not checked

(errcheck)